cs-date-time/src/main/java/io/cloudslang/content/datetime/utils/DateTimeUtils.java

```diff
@@ -0,0 +1,107 @@
+package io.cloudslang.content.datetime.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+
+/**
+ * Created by ursan on 7/6/2016.
+ */
+public class DateTimeUtils {
+    public static boolean isUnix(String locale) {
+        return Constants.Miscellaneous.UNIX.equals(locale);
+    }
+    public static boolean isMilliseconds(String locale) {
+        return Constants.Miscellaneous.MILLISECONDS.equals(locale);
+    }
+    /**
+     * Because the date passed as argument can be in java format, and because not all formats are compatible
+     * with joda-time, this method checks if the date string is valid with java. In this way we can use the
+     * proper DateTime without changing the output.
+     *
+     * @param date date passed as argument
+     * @return true if is a java date
+     */
+    public static boolean isDateValid(String date, Locale locale) {
+        try {
+            DateFormat format = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, locale);
+            format.parse(date);
+            return true;
+        } catch (ParseException e) {
+            return false;
+        }
+    }
+
+    /**
+     * Generates the locale
+     *
+     * @param lang    the language
+     * @param country the country
+     * @return the locale
+     */
+    public static Locale getLocaleByCountry(String lang, String country) {

```

Comment:
Shouldn't the lang parameter have a @notnull annotation? You can use javax.validation.constraints.NotNull to not add another maven dependency or use the one from intellij.